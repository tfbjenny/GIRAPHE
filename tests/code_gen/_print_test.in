print(23);
print(-1.2);
print(1>2);
print(true);
print("Hello World!");
print(null);
print(node("a"));
print([1, 2, 3]);
print({"a": 1, "b": 2});
print(1, true, "Hello~");

int a = 1;
float b = 1.2;
string d = "What!";

printf("%d--\n%.2f--\n%s\n", a, b , d);
