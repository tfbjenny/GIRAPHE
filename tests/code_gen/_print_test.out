23
-1.200000
false
true
Hello World!
null
node  0: a
list:[1, 2, 3]
{b: 2, a: 1}
1
true
Hello~
1--
1.20--
What!
